lib/my_spawn.ts

```diff
@@ -0,0 +1,39 @@
+import * as child_process from 'child_process';
```

Comment:
Nit: could the file name be something either more generic like "util.ts"? Just in case we decide to throw other methods in here that are not just for spawn.